docs/organizations/managing-security-and-risk.md

```diff
@@ -1,73 +1,129 @@
+
+### Top 10 high-risk repositories
+
+The **Top 10 high-risk repositories** list shows the repositories with the highest number of open findings, ordered by severity.
+
+!!! note
+    This panel may list fewer than ten repositories if there are fewer than ten repositories with open findings in the organization or if fewer than ten repositories are selected in the dropdown **Repository** filter.
+
+![Security and risk management high risk repositories panel](images/security-risk-management-overview-top-risk.png)
+
+### Top 10 common security categories
+
+The **Top 10 common security categories** list shows the most common security categories of open findings, ordered by count.
+
+To access the findings page with the corresponding filter applied, click on a category.
+
+![Security and risk management top categories panel](images/security-risk-management-overview-top-categories.png)
+
+## Findings {: id="item-list"}
+
+The **Security and risk management findings** page displays a filtered list of findings. By default, this list is sorted by status, and you can click the **First detected** column name to sort the findings by the detection date. Use this page to review and prioritize findings and track the progress of your security efforts.
+
+To access the findings page, access the [overview page](#dashboard) and click the **Findings** tab.
+
+![Security and risk management findings page](images/security-risk-management-findings.png)
+
+When viewing the findings, you can update the filtering criteria by clicking the **Severity**, **Status**, **Repository**, or **Security category** drop-downs above the list.
+
+To find out more about a finding in the list, click its **Details** column to navigate to the finding details on the source platform. Click **View all findings** to get back to the findings list.
+
+![Security and risk management finding details](images/security-risk-management-finding-details.png)
+
+## Sharing a filtered view of findings {: id="sharing-filtered-view"}
+
+To share the current view of the overview or findings page, click the **Copy URL** button in the top right-hand corner of the page. This action copies the URL with the current filters applied to the clipboard.
+
+## Exporting findings {: id="exporting-the-security-item-list"}
 
 !!! info "This feature is available only to organization admins and organization managers"
 
-To export a list of security items as a CSV file, click the **Export CSV** button in the top right-hand corner of the page. The exported list always includes all items, ignoring any applied filters.
+To export a list of findings as a CSV file, click the options menu in the top right-hand corner of the page and select **Export findings (.csv)**. The exported list always includes all findings, ignoring any applied filters.
+
+## Reviewing severity rules and integration settings {: id="reviewing-settings"}
```

Comment:
@nicklem I find this section a bit confusing...

1. **Review** the severity rules gives me the expectation that I can update the defined rules somehow. However, it doesn't seem possible, but I don't have any information about it in this section, or in [the linked section](https://github.com/codacy/docs/pull/2052/files#diff-30a735ab496ba1650575435a2195b32444487dbb269bb4b685b2e1ad8c7ebe53R202).
2. For the Jira integration settings, the link takes me to the integration installation docs, where I don't find any details about the integration settings.
3. Also, when I click the link **View integrations** on the **Security and risk** screen menu, I'm redirected to the Git provider integration settings... (more confused!)

Reply:
@claudiacarpinteiro I [tweaked](https://github.com/codacy/docs/pull/2052/commits/22656af7a6f315e59e8677deb01ccd2dab2a01f5) to address your points.

It makes sense to update the CTA on the button from "View integrations" to "Manage Jura integration", since it's the only third-party SRM integration for now.

<img width="238" alt="image" src="https://github.com/codacy/docs/assets/11616750/523ba413-52dc-4602-86a0-8bc97c7801e7">

Thoughts?

cc @joanasteodoro @alerizzo

Reply:
don't forget about slack! 🙌 that's currently a "security-only" integration right now

Reply:
Fair enough, although there's a mismatch with the context the user is in (the SRM page). The integration does push data to Slack, but has no immediate effect on the SRM dashboards.

Let's stick with the current CTA, "View integrations". I'll revert my suggestion.